app/gui/view/graph-editor/src/component/node.rs

```diff
@@ -923,3 +927,44 @@ impl display::Object for Node {
         &self.model.display_object
     }
 }
+
+
+// === Positioning ===
+
+/// Calculate a position where to render the [`background::View`] of a node, relative to the node's
+/// origin.
+///
+/// See the [`Node`] documentation for more details.
+fn background_offset(node_width: f32) -> Vector2 {
+    Vector2(node_width / 2.0, 0.0)
+}
+
+/// Calculate a position where to render the [`visualization::Container`] of a node, relative to
+/// the node's origin.
+///
+/// See the [`Node`] documentation for more details.
+fn visualization_offset(node_width: f32) -> Vector2 {
+    background_offset(node_width) + Vector2(0.0, VISUALIZATION_OFFSET_Y)
+}
+
+fn bounding_box(
+    (node_position, node_size, visualization_enabled_and_visible, visualization_size): &(
+        Vector2,
+        Vector2,
+        bool,
+        Vector2,
+    ),
+) -> BoundingBox {
```

Comment:
This was done on my review suggestion, and we do have precedence for it in `lib/rust/ensogl/component/selector/src/bounds.rs:63` (and some other functions in that file. 

This is a method that is only ever used in an FRP node, so this signature simplifies the call site to `bbox_input.map(bounding_box);` instead of requiring all parameters to be unpacked.

Reply:
@MichaelMauderer Based on a discussion with @wdanilo, I understand that the FRP API is expected to be able to automatically destructure tuples in the future, so we want to avoid a temporary workaround of tuple destructuring in function signatures.